sources/packages/web/src/components/common/LocationSummary.vue

```diff
@@ -24,14 +24,14 @@
     class="ma-2"
   >
     <v-row>
-      <v-col cols="10">
+      <v-col md="10" lg="10" xl="10">
```

Comment:
What is the difference between the code that was there before and the one you changed it to since all the various screen sizes use the same value ?

Reply:
as discussed in the call, it is for the breakpoints